go-update/src/main/java/org/reactome/release/goupdate/GoTermInstanceModifier.java

```diff
@@ -318,38 +343,33 @@ public void deleteGoInstance(Map<String, Map<String,Object>> goTerms, Map<String
 	
 	private void pointAllReferrersToOtherInstance(GKInstance replacementGOTerm) throws Exception
 	{
-//		Collection<GKInstance> referrers = GoTermInstanceModifier.getReferrersForGoTerm(this.goInstance);
-//		for (GKInstance referrer : referrers)
-//		{
+		@SuppressWarnings("unchecked")
+		Collection<GKSchemaAttribute> attributes = (Collection<GKSchemaAttribute>) this.goInstance.getSchemClass().getReferers();
```

Comment:
It's not intuitive to me that the referers of the goInstance would be 'attributes'. Doesn't 'getReferers' return 'instances' that refer to the current instance?

Reply:
In the context `this.goInstance.getSchemClass().getReferers()`, the function `getSchemaClass` will return the data model class for _this_ instance. Then, `getReferrers` is called on the schema class (_not_ on the instance), so it will return the referrer Schema Attributes for the Schema Class. The key thing is: `getReferrers` is not being called on `goInstance`, it is being called on `goInstance`'s Schema Class.